docs/testing.md

```diff
@@ -32,27 +32,80 @@ Those can be ignored and `/skip`ed. In example:
 ```
 Upstream issue: https://github.com/openshift/ci-operator-prowgen/issues/79 
 
+## Verify 
+
+This repository runs multiple checks to **verify** if it complies with standard go conventions.  The scripts from the target run through a series of checks to ensure the repository remains compliant with standard go practices. This can be ran **locally** with the command `make verify`.  When creating a pull-request, this tests is part of the acceptance criteria. 
```

Comment:
```suggestion
This repository runs a series of checks to **verify** if it complies with standard Golang conventions and practices.  This can be run **locally** with the command `make verify`.  When creating a pull-request, this test is part of the acceptance criteria. 
```